website/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "setup-typedoc": "ts-node setupTypedoc.ts",
+    "setup-typedoc": "ts-node ./src/scripts/setupTypedoc.ts",
+    "setup-soliditydocs": "ts-node ./src/scripts/setupSolidityDocs.ts",
```

Comment:
Noticed trying to run `npm run build` without first running this cause my build to fail:

```
~/code/pse/maci/website on  docs/contracts [$!?] via ⬢ v18.18.2
➜ npm run build

> website@0.0.0 build
> docusaurus build

[INFO] [en] Creating an optimized production build...
Warn: `docsDir` doesn't exist: "/Users/samrichards/code/pse/maci/website/docs".



✔ Client


✖ Server
  Compiled with some errors in 16.84s

Error: Docusaurus server-side rendering could not render static page with path /solidity-docs/TopupCredit because of error: IERC20 is not defined
    at serverEntry_render (main:72233:358)
    at runNextTicks (node:internal/process/task_queues:60:5)
    at process.processImmediate (node:internal/timers:447:9)
    at async /Users/samrichards/code/pse/maci/website/node_modules/p-map/index.js:57:22 {
  [cause]: ReferenceError: IERC20 is not defined
      at _createMdxContent (main:3906:98)
      at MDXContent (main:3936:8)
      at Uc (main:120769:44)
      at Xc (main:120771:253)
      at Z (main:120777:89)
      at Xc (main:120775:231)
      at Z (main:120777:89)
      at Xc (main:120771:481)
      at Z (main:120777:89)
      at Xc (main:120771:481)
```

I wonder, should we just run these two scripts as part of the `build` script? e.g.
```
"build": "npm run setup-typedoc && npm run setup-soliditydocs && docusaurus build",
```
Unsure if that's a good best practice, but it would ensure we avoid that issue.

Reply:
great catch, I think running all of it in the build script is fine!